From 56e15953da133c43580ce79130abf2cf41dddbe7 Mon Sep 17 00:00:00 2001
From: liyong4x <yongx.a.li@intel.com>
Date: Tue, 12 Jun 2018 21:25:58 -0400
Subject: [PATCH 098/129] media: intel-ipu4: at0231at: fix the image horizontal
 lines

Message for Open Source:
Modify the frame_length_line and line_length_pclk to
fix the issue that there're horizontal lines on the
dumped images in 2-HDR mode and 3-HDR mode.

Message for Internal:
Modify the frame_length_line and line_length_pclk to
fix the issue that there're horizontal lines on the
dumped images in 2-HDR mode and 3-HDR mode.

[Issue/Feature] There're horizontal lines on the dumped
images.

[Root Cause/Changes] Modify the fll and llp in mode
2-HDR,3-HDR,4-HDR initial settings.

Change-Id: I870a1927081b0ebc39336531df555da580a0b993
Tracked-On: #JIIAP-475
Signed-off-by: liyong4x <yongx.a.li@intel.com>
Signed-off-by: Meng Wei <wei.meng@intel.com>
---
 drivers/media/i2c/crlmodule/crl_ar0231at_configuration.h | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/drivers/media/i2c/crlmodule/crl_ar0231at_configuration.h b/drivers/media/i2c/crlmodule/crl_ar0231at_configuration.h
index 3b35854..bd17255 100644
--- a/drivers/media/i2c/crlmodule/crl_ar0231at_configuration.h
+++ b/drivers/media/i2c/crlmodule/crl_ar0231at_configuration.h
@@ -712,8 +712,8 @@ struct crl_ctrl_data_pair ar0231at_ctrl_data_modes[] = {
 	{ 0x3404, CRL_REG_LEN_16BIT, 0x0970, 0x10 },
 	{ 0x3082, CRL_REG_LEN_16BIT, 0x0004, 0x10 },
 	{ 0x30BA, CRL_REG_LEN_16BIT, 0x11F1, 0x10 },
-	{ 0x300C, CRL_REG_LEN_16BIT, 0x07BA, 0x10 },
-	{ 0x300A, CRL_REG_LEN_16BIT, 0x05C8, 0x10 },
+	{ 0x300C, CRL_REG_LEN_16BIT, 0x0872, 0x10 },
+	{ 0x300A, CRL_REG_LEN_16BIT, 0x054A, 0x10 },
 	{ 0x3042, CRL_REG_LEN_16BIT, 0x0000, 0x10 },
 	{ 0x3238, CRL_REG_LEN_16BIT, 0x0222, 0x10 },
 	{ 0x3238, CRL_REG_LEN_16BIT, 0x0222, 0x10 },
@@ -1051,8 +1051,8 @@ struct crl_ctrl_data_pair ar0231at_ctrl_data_modes[] = {
 	{ 0x3404, CRL_REG_LEN_16BIT, 0x0970, 0x10 },
 	{ 0x3082, CRL_REG_LEN_16BIT, 0x0008, 0x10 },
 	{ 0x30BA, CRL_REG_LEN_16BIT, 0x11F2, 0x10 },
-	{ 0x300C, CRL_REG_LEN_16BIT, 0x07BA, 0x10 },
-	{ 0x300A, CRL_REG_LEN_16BIT, 0x05C8, 0x10 },
+	{ 0x300C, CRL_REG_LEN_16BIT, 0x0872, 0x10 },
+	{ 0x300A, CRL_REG_LEN_16BIT, 0x054A, 0x10 },
 	{ 0x3042, CRL_REG_LEN_16BIT, 0x0000, 0x10 },
 	{ 0x3238, CRL_REG_LEN_16BIT, 0x0222, 0x10 },
 	{ 0x3238, CRL_REG_LEN_16BIT, 0x0222, 0x10 },
@@ -1390,8 +1390,8 @@ struct crl_ctrl_data_pair ar0231at_ctrl_data_modes[] = {
 	{ 0x3404, CRL_REG_LEN_16BIT, 0x0970, 0x10 },
 	{ 0x3082, CRL_REG_LEN_16BIT, 0x000C, 0x10 },
 	{ 0x30BA, CRL_REG_LEN_16BIT, 0x11F3, 0x10 },
-	{ 0x300C, CRL_REG_LEN_16BIT, 0x08C6, 0x10 },
-	{ 0x300A, CRL_REG_LEN_16BIT, 0x0518, 0x10 },
+	{ 0x300C, CRL_REG_LEN_16BIT, 0x09B8, 0x10 },
+	{ 0x300A, CRL_REG_LEN_16BIT, 0x0498, 0x10 },
 	{ 0x3042, CRL_REG_LEN_16BIT, 0x0000, 0x10 },
 	{ 0x3238, CRL_REG_LEN_16BIT, 0x0222, 0x10 },
 	{ 0x3238, CRL_REG_LEN_16BIT, 0x0222, 0x10 },
-- 
1.9.1

